artfiles.py

```diff
@@ -5,63 +5,73 @@
 # Copyright (c) 2016, InnoGames GmbH
 #
 
-import time
-import sys
-import urllib
-import argparse
-from socket import gethostname
+from time import time
+from urllib import urlopen
+from argparse import ArgumentParser
 
-parser = argparse.ArgumentParser()
-parser.add_argument(
-    "-u",
-    "--user",
-    dest="http_user",
-    type=str,
-    required=True,
-    help="the http user name to authenticate")
-parser.add_argument(
-    "-p",
-    "--pass",
-    dest="http_pass",
-    type=str,
-    required=True,
-    help="the http password to authenticate")
-args = parser.parse_args()
 
+def parse_args():
+    parser = ArgumentParser()
+    parser.add_argument(
+        "-u",
+        "--user",
+        dest="http_user",
+        type=str,
+        required=True,
+        help="the http user name to authenticate")
+    parser.add_argument(
+        "-p",
+        "--pass",
+        dest="http_pass",
+        type=str,
+        required=True,
+        help="the http password to authenticate")
+    return parser.parse_args()
 
-artfiles_url = 'https://%s:%s@dcp.c.artfiles.de/api/stats/get_estats.html' % (
-    args.http_user, args.http_pass)
-data = urllib.urlopen(artfiles_url)
-csv = data.readlines()
 
-now = str(int(time.time()))
+def main():
+    args = parse_args()
+    artfiles_url = (
+        'https://{}:{}@dcp.c.artfiles.de/api/stats/get_estats.html'
```

Comment:
I prefer using named parameters, because one can directly see which values should go in there.